building.rb

```diff
@@ -1,2 +1,39 @@
 class Building
+
+  attr_reader :address, :apartments
+
+  def initialize(args= {})
+    @address = args.fetch(:address, "")
+    @apartments = args.fetch(:apartments, "")
+  end   
+
+  def total_room_count
+    @apartments.inject(0) {|sum, apt| sum + apt.room_count } 
+  end   
+
+  def total_monthly_revenue
+    @apartments.inject(0) {|sum, apt| sum + apt.monthly_rent } 
+  end 
+    
+  def apartments_by_rent
+    @apartments.sort_by {|apt| apt.monthly_rent}.reverse
```

Comment:
Another way to solve this would be by using 
```ruby
apartments.sort {|apartment1, apartment2| apartment2.monthly_rent <=> apartment1.monthly_rent}
```
which would eliminate the need to call 2 methods on @apartments. It would sort it all at once in descending order rather than sorting it in ascending order and then reversing it. 
@kimberlysuazo